po/be.po

```diff
@@ -0,0 +1,937 @@
+#, javascript-format
+msgid "%s・Other"
+msgstr "%s・Іншы"
+
+#. TRANSLATORS: A fax number
+#: src/service/ui/contacts.js:123
+#, javascript-format
+msgid "%s・Fax"
+msgstr "%s・Факс"
+
+#. TRANSLATORS: A work phone number
+#: src/service/ui/contacts.js:127
+#, javascript-format
+msgid "%s・Work"
+msgstr "%s・Работа"
+
+#. TRANSLATORS: A mobile or cellular phone number
+#: src/service/ui/contacts.js:131
+#, javascript-format
+msgid "%s・Mobile"
+msgstr "%s・Мабільны"
+
+#. TRANSLATORS: A home phone number
+#: src/service/ui/contacts.js:135
+#, javascript-format
+msgid "%s・Home"
+msgstr "%s・Дом"
+
+#. TRANSLATORS: A phone number (eg. "Send to 555-5555")
+#: src/service/ui/contacts.js:278 src/service/ui/contacts.js:292
+#, javascript-format
+msgid "Send to %s"
+msgstr "Адправіць у %s"
+
+#. TRANSLATORS: Title of keyboard shortcut dialog
+#: src/service/ui/keybindings.js:33
+msgid "Set Shortcut"
+msgstr "Задаць спалучэнні клавіш"
+
+#. TRANSLATORS: Button to confirm the new shortcut
+#: src/service/ui/keybindings.js:47
+msgid "Set"
+msgstr "Задаць"
+
+#. TRANSLATORS: Summary of a keyboard shortcut function
+#. Example: Enter a new shortcut to change Messaging
+#: src/service/ui/keybindings.js:54
+#, javascript-format
+msgid "Enter a new shortcut to change <b>%s</b>"
+msgstr "Увядзіце новыя спалучэнні, каб змяніць <b>%s</b>"
+
+#. TRANSLATORS: Keys for cancelling (␛) or resetting (␈) a shortcut
+#: src/service/ui/keybindings.js:83
+msgid "Press Esc to cancel or Backspace to reset the keyboard shortcut."
+msgstr "Націсніце Esc, каб скасаваць або Прабел, каб скінуць спалучэнне клавіш."
+
+#. TRANSLATORS: When a keyboard shortcut is unavailable
+#. Example: [Ctrl]+[S] is already being used
+#: src/service/ui/keybindings.js:182
+#, javascript-format
+msgid "%s is already being used"
+msgstr "%s ужо выкарыстоўваецца"
+
+#: src/service/ui/service.js:31
+msgid "Select a Device"
+msgstr "Выбраць прыладу"
+
+#: src/service/ui/service.js:35
+msgid "Select"
+msgstr "Выбраць"
+
+#: src/service/ui/settings.js:443
+msgid "Unpaired"
+msgstr "Разлучана"
+
+#: src/service/ui/settings.js:445
+msgid "Disconnected"
+msgstr "Адлучана"
+
+#: src/service/ui/settings.js:448
+msgid "Connected"
+msgstr "Падлучана"
+
+#: src/service/ui/settings.js:516
+#, javascript-format
+msgid "Transferred files are placed in the <a href=\"%s\">Downloads</a> folder."
+msgstr "Перанесеныя файлы размяшчаюцца ў папцы <a href=\"%s\">Спампоўкі</a>."
+
+#: src/service/ui/settings.js:607
+msgid "A complete KDE Connect implementation for GNOME"
+msgstr "Паўнавартасная рэалізацыя KDE Connect для GNOME"
+
+#. TRANSLATORS: eg. 'Translator Name <your.email@domain.com>'
+#: src/service/ui/settings.js:616
+msgid "translator-credits"
+msgstr "Maksim Krapiŭka <metalomaniax@gmail.com>"
+
+#: src/service/ui/settings.js:646
+msgid "Debug Logging Enabled"
+msgstr "Журнал адладкі ўключаны"
+
+#: src/service/ui/settings.js:647
+msgid "Reproduce the problem and press “Done” to review the log."
```

Comment:
I think this is still from the previous POT file. Maybe after you've fetched and rebased you should update with ninja?

    ninja -C _build org.gnome.Shell.Extensions.GSConnect-update-po